packages/shoreline/src/components/collection/collection-view.tsx

```diff
@@ -74,7 +74,12 @@ export const CollectionView = forwardRef<HTMLDivElement, CollectionViewProps>(
           <EmptyStateIllustration data-sl-collection-view-illustration>
             {getIcon(status)}
           </EmptyStateIllustration>
-          <Heading data-sl-collection-view-heading>{heading}</Heading>
+          <Heading
+            data-sl-collection-view-heading
+            style={{ textAlign: 'center' }}
```

Comment:
you should add this style to the `collection-view.css` [theme file](https://github.com/vtex/shoreline/blob/main/packages/shoreline/src/themes/sunrise/components/collection-view.css)

Something like that:
```css
[data-sl-collection-view-heading] {
  text-align: center;
}
```

Reply:
Uuuh! Good to know about this file <3